firecrown/loglike.py

```diff
@@ -20,9 +20,10 @@ def compute_loglike(*, cosmo, data):
     """
     loglike = 0.0
     statistics = {}
+
     analyses = list(
         set(list(data.keys())) -
-        set(['parameters', 'run_metadata']))
+        set(['parameters', 'run_metadata', 'sampler']))
```

Comment:
Ack I forgot to remove `run_metadata` from this last PR. My bad. I am happy to submit a PR to clean it out or if you have a chance you can just delete it.

Reply:
removed.